demisto_sdk/commands/lint/linter.py

```diff
@@ -783,7 +783,7 @@ def _docker_run_pytest(self, test_image: str, keep_container: bool, test_xml: st
         try:
             # Running pytest container
             cov_file_path = Path.joinpath(self._pack_abs_dir, '.coverage')
-            cov = self._pack_abs_dir.stem if not no_coverage and cov_file_path.exists() else ''
+            cov = self._pack_abs_dir.stem if not no_coverage else ''
```

Comment:
i'd reverse it, (`'' if no coverage else ...`), but it's a matter of taste.

Reply:
also, do we intentionally remove the `and cov_file_path.exists()`?

Reply:
yes, pytest will create this file